cms/djangoapps/contentstore/views/course.py

```diff
@@ -1262,8 +1196,8 @@ def textbooks_list_handler(request, course_key_string):
             existing = course.pdf_textbooks
             existing.append(textbook)
             course.pdf_textbooks = existing
-            if not any(tab['type'] == PDFTextbookTabs.type for tab in course.tabs):
-                course.tabs.append(PDFTextbookTabs())
+            if not any(tab['type'] == 'pdf_textbooks' for tab in course.tabs):
+                course.tabs.append(CourseTab.from_json({'type': 'pdf_textbooks'}))
```

Comment:
Nit: it looks like there's a lot of duplicated code in PUT vs POST. Is it worth abstracting, perhaps by using a class-based view?